### ISSUE TITLE
message-aria-describedby should also work for v3 web components

### ISSUE THREAD

**Issue by rhasselle-oddball** [department-of-veterans-affairs/vets-design-system-documentation#1893]
# Bug Report

- [x] I’ve searched for any related issues and avoided creating a duplicate issue.

## What happened

`message-aria-describedby` works for v1 web component `va-text-input`, but not v3 (uswds=true). Tested using the react binding version of web components by passing `messageAriaDescribedby`

## What I expected to happen

Using v3 web components, `message-aria-describedby` should work propertly. Should work for native web component (using `message-aria-describedby`, and also react binding (using `messageAriaDescribedby`). 

## Reproducing
Go to story book and try on v1 web component, and v3 web component, and notice `message-aria-describedby` doesn't work for v3 web component

## Urgency
How urgent is this request? Please select the approriate option below and/or provide details
- [ ] This bug is blocking work currently in progress
- [ ] This bug is affecting work currently in progress but we have a workaround
- [x] This bug is blocking work planned within the next few sprints
- [ ] This bug is not blocking any work
- [ ] Other

### Related tickets
- https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1852
- https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1852
